spark/src/main/scala/geotrellis/spark/io/hadoop/HadoopAttributeStore.scala

```diff
@@ -27,10 +27,16 @@ class HadoopAttributeStore(val hadoopConfiguration: Configuration, attributeDir:
     new Path(attributeDir, fname)
   }
 
+  private def _delete(layerId: LayerId, path: Path): Unit = {
+    if(!layerExists(layerId)) throw new LayerNotFoundError(layerId)
+    HdfsUtils
+      .listFiles(new Path(attributeDir, path), hadoopConfiguration)
```

Comment:
It doesn't support globing ): tried it in late commits, then during not working test found that attributes are not deleting.